VideoWeb/VideoWeb/ClientApp/src/app/shared/self-test/self-test.component.scss

```diff
@@ -7,3 +7,8 @@
   color: govuk-colour('light-blue');
   text-decoration-line: underline;
 }
+
+.self-view-container {
+  width: 270px;
```

Comment:
We now have this width defined in 2 places if it ever needs to be changed. Maybe setting the element width in the html to 100% would help?

Reply:
I have fixed this to be 100%. Didn't actually think it would work before which is why I left it. Good shout